proto/workflow_execution_service.proto

```diff
@@ -91,15 +92,15 @@ message workflow_request {
 
   // OPTIONAL
   // A key-value map of arbitrary metadata outside the scope of the workflow_params but useful to track with this workflow request
-  map<string,string> key_values = 5;
+  map<string,string> tags = 5;
```

Comment:
Do we want to add a parameter that facilitates the ingestion of additional files defined in the workflow runtime-descriptor <json> file ?